main.go

```diff
@@ -14,7 +15,19 @@ import (
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
 )
 
-var svc = &dynamodb.DynamoDB{}
+const (
+	usersTable = "users"
```

Comment:
in this specific example, I actually created a table called "basic-auth".  if we change it to "users" the plugin won't work out of the box

Reply:
this is a PoC - so it doesn't matter what the table is called. either configurable, or hardcode it. but is a draft PR - so was planning on making it configurable - eventually.